### PR TITLE
Feature sprint09 tijn percent based base level change

### PR DESCRIPTION
**Pull request by tschmits**

### CODE REVIEW

File: app/scripts/components/scenariobuilder.js [hunk 1]
```diff
@@ -424,6 +421,37 @@ var exports = (function() {
         });
 
         return scenario;
+      },
+
+      // multiplytable methods
+
+      collapseToggle: function (e) {
+        e.stopPropagation();
```
Comment (SiggyF) [inline app/scripts/components/scenariobuilder.js]: `stop` is already defined in click attribute

File: app/scripts/components/scenariobuilder.js [hunk 2]
```diff
@@ -424,6 +421,37 @@ var exports = (function() {
...
+        $(e.target).parent(".multiplytable").children(".collapse").collapse("toggle");
+      },
+
+      split: function (string) {
```
Comment (SiggyF) [inline app/scripts/components/scenariobuilder.js]: `_.split`?

File: app/scripts/components/scenariobuilder.js [hunk 3]
```diff
@@ -424,6 +421,37 @@ var exports = (function() {
...
+      },
+
+      getVar: function (id) {
+        return _.filter(
+          _.flattenDeep(
+            _.map(this.scenarioConfig.sections, function (section) {
+              return section.variables;
+            })
+          ),
+          function (variable) {
```
Comment (SiggyF) [inline app/scripts/components/scenariobuilder.js]: Same as `['id', id]`?

File: app/scripts/components/scenariobuilder.js [hunk 4]
```diff
@@ -424,6 +421,37 @@ var exports = (function() {
...
+            return variable.id === id;
+          }
+        )[0];
+      },
+
+      calcAbsBaseLevelChange: function (basinslope, percentage) {
+        return _.round(
+          ((4 + (10000 * Math.tan(basinslope / 180 * Math.PI))) * percentage / 100),
```
Comment (SiggyF) [inline app/scripts/components/scenariobuilder.js]: Explain formula?

File: app/styles/scenariobuilder.scss [hunk 5]
```diff
@@ -33,6 +33,35 @@
     padding: 8px 16px; // Extra pixel top and bottom for border on buttons
    }
 
+  .toggle {
+    margin-top: $padding;
+  }
+
+  .multiplytable {
```
Comment (SiggyF) [inline app/styles/scenariobuilder.scss]: If the goal is to make something full-width, maybe rename this to something like "full-width"?

File: gulpfile.babel.js [hunk 6]
```diff
@@ -27,7 +27,7 @@ const reload = browserSync.reload;
 var args = require("yargs").argv;
 // Server used for serving remote url"s
 // "http://136.231.10.175:8888";
-var apiServer = "https://delft3dgt-nginx";
+var apiServer = "http://delft3dgt-django:8000";
```
Comment (SiggyF) [inline gulpfile.babel.js]: conflicts with other PR

File: app/scripts/components/scenariobuilder.js (hunk 3 above)
Comment (tschmits) [inline app/scripts/components/scenariobuilder.js]: What do you mean? I don't understand.

File: app/styles/scenariobuilder.scss (hunk 5 above)
Comment (tschmits) [inline app/styles/scenariobuilder.scss]: Hmmm should do, but the goal is not to make it full-width. The component multiplytable is a Vue generated div with computed values. The btn which toggles this table should be padded top and bottom. That's why this scss is here.

**Pull request merged**